cli/azd/cmd/middleware/middleware.go

```diff
@@ -86,7 +86,7 @@ func (r *MiddlewareRunner) RunAction(
 	// allowing the middleware to choose to execute logic before and/or after
 	// the action. After we have executed all of the middlewares the action is run
 	// and the chain is unwrapped back out through the call stack.
-	nextFn = func(nextContext context.Context) (*actions.ActionResult, error) {
+	nextFn = func(ctx context.Context) (*actions.ActionResult, error) {
```

Comment:
@vhvb1989 pointed out that this diff was a little weird.  The issue was that L109 was calling run but passing in the "ctx" not "nextContext".  It would have also been correct to not rename this variable and instead edit L109 so that we called `action.Run(nextContext)` instead, but I figured shadowing here was better.  In general, we don't want to be using any other context except the one that was passed into this next function.